.flake8

```diff
@@ -1,3 +1,5 @@
 [flake8]
-max-line-length = 120
-ignore = E501
+ignore = E203, E266, E501, W503
```

Comment:
FWIW it looks like this file supports `# comments`. Could specify what the `ignored` and `select` values mean

See examples in http://flake8.pycqa.org/en/latest/user/configuration.html

Reply:
Thanks. Will do.

Reply:
6360452